src/Extension/FluentExtension.php

```diff
@@ -965,19 +964,24 @@ protected function getRecordLocale()
         return Locale::getCurrentLocale();
     }
 
-
     /**
      * Returns the source locale that will display the content for this record
      *
      * @return Locale|null
      */
-    public function getSourceLocale()
+    public function getSourceLocale(): ?Locale
```

Comment:
```suggestion
    public function getSourceLocale()
```
as per https://github.com/tractorcow-farm/silverstripe-fluent/pull/840#discussion_r1556482111

Reply:
whoops, forgot about this, will make the change